plugins/module_utils/oneview.py

```diff
@@ -330,6 +330,10 @@ def compare(first_resource, second_resource, parameter_to_ignore=None):
                 # Inexistent key is equivalent to exist with value None
                 logger.debug("%s %s", OneViewModuleBase.MSG_DIFF_AT_KEY.format(
                     key), debug_resources)
+                if parameter_to_ignore is not None:
```

Comment:
As discussed ignore can be put just after the for loop. Not require go so inside

Reply:
I have changed it per our discussion. Thanks Avijit